lms/templates/courseware/course_about.html

```diff
@@ -319,8 +319,16 @@
             </li>
           % endif
 
+          % if pre_requisite_courses:
+          <li class="prerequisite-course important-dates-item">
+            <i class="icon fa fa-list-ul"></i>
+            <p class="important-dates-item-title">${_("Prerequisites")}</p>
+            <span class="important-dates-item-text pre-requisite">${pre_requisite_courses[0]}</span>
```

Comment:
nitpick: for now we might want a comment to explain why we're just dereferencing the first element